functional-samples/cookbook.push/background.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code is a modified version of
+// https://codelabs.developers.google.com/codelabs/push-notifications
+
+// These methods are meant to be called by the developer following the cookbook.
+/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "subscribeUser" }]*/
+
+const applicationServerPublicKey = '<key>';
+
+// Whether the extension will attempt to show a notification or not. Controlled
+// by subscribeUserVisibleOnlyFalse() and subscribeUserVisibleOnlyTrue(). Not
+// meant to be used in production due to ephemeral nature of service workers.
+let should_show_notification = true;
+
+function subscribeUser(userVisibleOnly) {
+  should_show_notification = userVisibleOnly;
```

Comment:
same comment about change snake case to camel case

Reply:
PTAL for camel case, I'm a little confused on where you use it. Only for local variables, but not for functions?

Reply:
Should just always be camel case, I’m not aware of any place that should be using snake case

Reply:
I believe this is addressed then. All variables/functions are camel case now.